src/App.js

```diff
@@ -3,14 +3,16 @@ import React from "react";
 import Footer from "./components/Footer/Footer";
 import { BrowserRouter as Router } from "react-router-dom";
 import Navbar from "./components/Navbar/Navbar";
+import ShopListItem from "./components/ShopListItem/ShoptListItem";
 
 function App() {
   return (
     <Router>
       <Navbar />
+      <ShopListItem/>
```

Comment:
this should take props
```suggestion
      <ShopListItem image={""} title={""} ..../>
```